### ISSUE TITLE
Can't trigger the subscription resolver

### ISSUE THREAD

**Issue by Cuxs** [apollographql/graphql-subscriptions#124]
I'm trying to make a simple chat, it is very similar that the one in the example, but I don't use the _graphql-tools_ library so there are a few things that I do differently and I don't know how they have to be done

My **schema**:
```javascript
const schema = new Schema({
  query: new GraphQLObjectType({...})
  mutation: new GraphQLObjectType({...})
  subscription: new GraphQLObjectType({
    name: 'Subscriptions',
    fields: {
      messageAdded,
    },
  }),
});
```
The **subscription** called _messageAdded_ it is defined as:
```javascript
 messageAdded: {
    name: 'messageAdded',
    type: MessageType,
    args: {
      messageThread_id: { type: new GraphQLNonNull(GraphQLInt) },
    },
    resolve: (payload, args, context, info) => {  // Without this it does not work neither
      return payload.messageAdded;
    },
    subscribe: () =>
      pubsub.asyncIterator('messageAdded')
  }
};
```

The **mutation** that triggers that subscription it's: 
```javascript
 addMessage: {
    type: MessageType,
    args: {
      from_id: { type: Int },
      content: { type: new NotNull(Gstring) },
      messageThread_id: { type: new NotNull(Int) },
    },
    resolve: (value, { from_id, content, messageThread_id }) =>
      //........
          return Message.create({
            from_id,
            content,
            messageThread_id,
          })
            .then((msg) => {
              pubsub.publish('messageAdded', { messageAdded: msg, messageThread_id: msg.messageThread_id });
               return msg;
            });
        }),
  },
```
Finally this is my **server** configuration:
```javascript
const ws = createServer(app);

SubscriptionServer.create({
  execute,
  subscribe,
  schema,
}, {
  server: ws,
  path: '/subscriptions',
});
app.use('/graphiql', graphiqlExpress({
  endpointURL: '/graphql',
  subscriptionsEndpoint: 'ws://localhost:4000/subscriptions',
}));
app.use('/graphql', bodyParser.json(), graphqlExpress({ schema }));

```

And this happend:
<img width="892" alt="screen shot 2018-01-05 at 14 57 06" src="https://user-images.githubusercontent.com/8927894/34621661-cb26f50a-f228-11e7-98b6-f1a642eb4ad0.png">
<img width="946" alt="screen shot 2018-01-05 at 14 56 51 1" src="https://user-images.githubusercontent.com/8927894/34621666-ce39aee0-f228-11e7-9013-9bcc20fb6833.png">

--Nothing--
Basically the problem is that the subscription defined as _messageAdded_ does not "catch" the mutation: it just stays with that text like waiting for the server publication. 

As you can see it's my try number 79 😞 
 I've been searching for a while, but I cannot understand what am I doing wrong.

Hope this is clearly enough to understand.

**Comment by Cuxs** [comment]
The mistake was in my server configuration. I have to listen with the ws instance
```javascript
const ws = createServer(app);

ws.listen(process.env.PORT || 4000, () => {
  console.log(`Running a GraphQL API server at http://localhost:${process.env.PORT || 4000}/graphql`);
  
  new SubscriptionServer.create({
    execute,
    subscribe,
    schema,
  }, {
    server: ws,
    path: '/subscriptions',
  });
});
```
not with the  app instance
```javascript
const app = express();
app.listen(4000)
app.listen(process.env.PORT || 4000); // WRONG
```

**Comment by Monteth** [comment]
Can I ask you how do you create PubSub instance?
